homeassistant/components/vivotek/__init__.py

```diff
@@ -1 +1,32 @@
 """The Vivotek camera component."""
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .camera import async_test_config
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = [Platform.CAMERA]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up the Vivotek component from a config entry."""
+
+    try:
+        await async_test_config(hass, entry.data)
```

Comment:
```suggestion
        entry.runtime_data = await async_test_config(hass, entry.data)
```
https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/runtime-data/